media/js/firefox/os/devices.js

```diff
@@ -76,12 +76,16 @@ if (typeof window.Mozilla === 'undefined') {
     /*
     * Track telecom provider link clicks/page exits in Google Analytics
     */
+
     var trackProviderExit = function(e) {
         var $this = $(this);
         var newTab = (this.target === '_blank' || e.metaKey || e.ctrlKey);
 
+        window.dataLayer = window.dataLayer || [];
         if (newTab) {
-            gaTrack(['_trackEvent', '/os/devices/ Interactions', 'Get a Phone Overlay Exits', $this.text()]);
+            window.dataLayer.push({
+                event: 'overlay-exit'
```

Comment:
Do we no longer need to track the name of the provider being clicked? If that's so, we could remove the `var $this = $(this);` variable declaration on line 81 as it's no longer used.